tests/input_files/sra_samples.tsv

```diff
@@ -0,0 +1,3 @@
+sample	fq1	fq2
```

Comment:
I'm not sure whether the user should have to know in advance how the samples table headers have to be called. Because if we assumed the same principle for the other independent pipeline - HTSinfer - the user would have to write the header for all parameters we are going to infer, before starting the pipeline (yes, of course we would provide a template...) 
...what I mean is, would it be nice to let the user supply the tsv with just one SRR per row and nothing else, and then add the header "sample\tfq1\tfq2" when loading the samples table?

Reply:
hm yeah that could also work. I was simply assuming that the sample table is there. Shouldn't be much work to just require column "sample" (with or without this header) and the SRR identifiers. 
But I'm reluctant to make changes before we settled on this.

Reply:
Thinking about this again I'd actually prefer not to have "fq1" and "fq2" in the header of the input samples table *for the sake of reusability*: Think about using this SRA workflow independent of  ZARP(-cli)... But still, it's not important, so good to merge from my side